docs/socket-types.md

```diff
@@ -0,0 +1,16 @@
+this.socket.on('connect', this.onSocketConnected());
+    this.socket.on('connect_error', this.onSocketConnectionError);
+    this.socket.on('connect_timeout', this.onSocketTimeout);
+    this.socket.on('disconnect', this.onSocketDisconnected());
+    this.socket.on('reconnect', this.onSocketReconnected());
+    this.socket.on('reconnecting', this.onSocketReconnecting);
+    // this.socket.on('reconnect_error', socketConnectionError);
+    this.socket.on('reconnect_failed', this.onSocketReconnectionFailed);
+    this.socket.on('reconnect_timeout', this.onSocketTimeout);
```

Comment:
The message does update on the scenario ; it is handled in the notification component via a switch case.